external/mmdetection/detection_tasks/apis/detection/inference_task.py

```diff
@@ -169,10 +169,10 @@ def _create_model(config: Config, from_scratch: bool = False):
             model = build_detector(model_cfg)
         return model
 
-
-    def _add_predictions_to_dataset(self, prediction_results, dataset, confidence_threshold=0.0):
+    def _add_predictions_to_dataset(self, prediction_results, dataset, confidence_threshold=0.0,
+                                    return_saliency_map=False):
         """ Loop over dataset again to assign predictions. Convert from MMDetection format to OTE format. """
-        for dataset_item, (all_results, feature_vector) in zip(dataset, prediction_results):
+        for dataset_item, (all_results, feature_info) in zip(dataset, prediction_results):
```

Comment:
maybe `features` instead of `feature_info`. Maybe size, shape, source, format can be named as `info`, but it's not your case

Reply:
Good point, fixed